src/databricks/labs/ucx/assessment/clusters.py

```diff
@@ -210,7 +226,7 @@ def _assess_policies(self, all_policices) -> Iterable[PolicyInfo]:
             except KeyError:
                 spark_version = None
             policy_name = policy.name
-            creator_name = policy.creator_user_name
+            creator_name = policy.creator_user_name or None
```

Comment:
i wonder if `or None` makes sense here

Reply:
My reasoning here:

 * We have the same code-path on other types[^1] that treats both `""` and `None` as missing.
 * For all practical purposes, if this is ever `""` then indeed the creator is missing.
 * Rather than have downstream usage deal with these (potentially) differently, normalise them during crawling so that `None` means missing and anything else is not missing.

An alternative would be to treat `""` as present and for consistency update the other crawlers to not warn if this were to ever occur.

Either way is fine, so long as we're consistent and the semantics are well-defined.

[^1]: `ClusterInfo`, `JobInfo`, `PipelineInfo`